IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayTrackStructurePart/DocPropertySet.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayTrackStructurePart" UniqueId="011923c1-fbf2-84f9-6215-e3ea43caebc9" ApplicableType="IfcFacilityPart/IfcRailwayPartTypeEnum(.TRACKSTRUCTURE.),IfcFacilityPart/IfcRailwayPartTypeEnum(.PLAINTRACKSUPESTRUCTURE.),IfcFacilityPart/IfcRailwayPartTypeEnum(.TURNOUTSUPERSTRUCTURE.),IfcFacilityPart/IfcRailwayPartTypeEnum(.DILATATIONSUPERSTRUCTURE.)" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayTrackStructurePart" UniqueId="011923c1-fbf2-84f9-6215-e3ea43caebc9" ApplicableType="IfcFacilityPart/IfcRailwayPartTypeEnum(.TRACKSTRUCTURE.),IfcFacilityPart/IfcRailwayPartTypeEnum(.PLAINTRACKSUPESTRUCTURE.),IfcFacilityPart/IfcRailwayPartTypeEnum(.TURNOUTSUPERSTRUCTURE.),IfcFacilityPart/IfcRailwayPartTypeEnum(.DILATATIONSUPERSTRUCTURE.)" PropertySetType="PSET_OCCURRENCEDRIVEN">
```

Comment:
These ApplicableType-strings would probably change after the redesign of facility part structure.